captainhook.template.json

```diff
@@ -61,6 +61,6 @@
         "actions": []
     },
     "config": {
-        "php-path": "/usr/bin/php"
+        "php-path": "/usr/bin/env php7.4"
```

Comment:
This should work. There is also path to php-cs-fixer in line 36 which we should also change.